src/components/ListItem.tsx

```diff
@@ -52,6 +52,20 @@ export function ListItemCheckBox({ item, listPath }: Props) {
 		}
 	};
 
+	const deleteItemHandler = () => {
+		const isConfirmed = window.confirm("Do you want to delete this item?");
+		console.log(isConfirmed);
+		console.log("Item:", item);
+		console.log("List Path:", listPath);
+
+		if (isConfirmed) {
+			deleteItem(listPath as string, item);
```

Comment:
Because `deleteItem` re-throws, we'll have to catch it here and handle it too 🙂

Reply:
Great point!
![image](https://github.com/user-attachments/assets/814573ef-6ade-4d0a-a6c7-1a2044426a3d)
We wrapped the deleteItem() func call in a try-catch block! In the catch, we have the error being logged into the console! @eternalmaha pointed out we can keep notifying the user simple by doing some sort of alert! I thought we could just tuck an alert in the catch block!

Reply:
@RossaMania and I were just thinking about the purposes of try-catchs in our deleteItem() call. I was wrapping my head around well "if deleteItem() throws an error as part of the function", do we need to throw an error again when its called? If im understanding correctly, its because there can be an error of course when deleting the item, then there can be an error when the function is being called itself. **as I research more about try-catches and function calls ** We decided to wrap a try catch around the whole function call.